tanglescope/blowballcollector.cpp

```diff
@@ -57,7 +58,10 @@ void BlowballCollector::analyzeBlowballsPeriodically() {
   auto& thisRef = *this;
   if (_snapshotInterval > 0) {
     pubWorker.schedule_periodically(pubThread.now(), std::chrono::seconds(_snapshotInterval),
-                                    [&thisRef, buckets](auto scbl) { thisRef.analyzeBlowballs(buckets); });
+                                    [&thisRef, buckets](auto scbl) {
```

Comment:
Unlike in C, in C++ you can just omit the variable name if you don't use it.
So you can do `[&thisRef, buckets](auto) {` instead and not having to use the `UNUSED` macro.
Same goes for all the following same cases.

Reply:
Got it. Modified all of them at https://github.com/iotaledger/entangled/pull/1297/commits/092494dbcb4156b95d4c4ddbb22b6ff058dd5a44